llvm/lib/Analysis/LoopAccessAnalysis.cpp

```diff
@@ -1646,22 +1644,19 @@ bool llvm::sortPtrAccesses(ArrayRef<Value *> VL, Type *ElemTy,
 
```

Comment:
might be good to split off the ::sortPtrAccesses changes separately, as this is what SLP uses I think